examples/fishnet_api/main.py

```diff
@@ -97,13 +97,43 @@ async def index():
 
 
 @app.get("/datasets")
-async def datasets() -> List[Dataset]:
-    return await Dataset.fetch_all()
+async def datasets(view_as: Optional[str], by: Optional[str]) -> str:
```

Comment:
The signature was changed. What I wanted is a `List` of all `Datasets`, which may have a `PermissionStatus` (depending on whether `view_as` is set), placed inside a `Tuple`.

Reply:
^- add this to every other endpoint where `.fetch_all()` is called